_posts/2018-03-15-usegalaxy-eu.md

```diff
@@ -0,0 +1,50 @@
+---
+tags: [devops]
+title: "Freiburg Galaxy becomes useGalaxy.EU"
+location: Freiburg, Germany
+site: freiburg
+supporters:
+ - denbi
+ - elixir
+ - ViCE
+---
+
+As part of our ongoing growth and development, Freiburg Galaxy is becoming UseGalaxy.eu
+
+## What Changes
+
+- You will be able to access the galaxy instance via [`https://usegalaxy.eu`](https://usegalaxy.eu) in addition to the old, `https://galaxy.uni-freiburg.de`.
+- Our website will read "UseGalaxy.eu" in many places where it used to say Freiburg Galaxy. However, this Galaxy is still primarily run and maintained by the Freiburg Galaxy Team.
+- You can reach us any time for questions at [contact@usegalaxy.eu](mailto:contact@usegalaxy.eu)
+
+## What Doesn't Change
+
+- Your data, the tools
+- The Freiburg Galaxy Team maintaing the instance and providing all of the training associated with our Galaxy.
+
+## The Future
+
+We are not trying to be the only Galaxy in all of Europe, but we believe we are
+in a good position to make our Galaxy and our services available to all of
+Europe, especially locations which may not have the expertise or resources to
+host their own Galaxy instance. Many times regional Galaxies will provide
+features that are desirable over using a main european one, such as
+localisation, support in your native language, and usage of local resources.
+
+We welcome other groups to join us in maintaining and improving the European
+Galaxy Instance, as part of that we have open sourced most of the tools used in
```

Comment:
most ? all?

Reply:
I know their is one repo that is still closed, but does it matter that much?

Reply:
-temporary is not opened up yet. (And never will be since it will be replaced with open-from-the-start official playbooks.)

Reply:
So, no, doesn't matter that much. All just feels disingenuous to say.